src/null-safety/migration-guide.md

```diff
@@ -430,34 +430,23 @@ If so, revert your code changes before using the migration tool again.
 We encourage you to publish packages as prereleases
 as soon as you migrate:
 
-* [Set the SDK constraints to the tested beta version.](#sdk-constraints)
 * [Set the package version to indicate a breaking change.](#package-version)
+* [Publish the package as a prerelease if development is still in progress.][]
```

Comment:
We should put the "if" part first (https://developers.google.com/style/clause-order). Also... maybe feature publishing first, and prereleases next?

```suggestion
* [Publish the package][publish].
  If you don't consider the migrated package to be final quality, then
  [publish it as a prerelease][publish-prerelease].
```

Of course you'd need to add/adjust the link definitions accordingly.